pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,542 @@
+      shibe, stop it fren big ol boof. Wow such tempt doge heckin good boys wow
+      very biscit heckin angery woofer he made many woofs, snoot heckin good
+      boys shoober wrinkler. You are doing me a frighten borkf ur givin me a
+      spook mlem vvv, much ruin diet heckin corgo.
+    </p>
+    <amp-story-player style="width: 370px; height: 622px;">
+      <script type="application/json">
+        {
+          "behavior": {
+            "autoplay": false
+          }
+        }
+      </script>
+
+      <a
+        href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/"
+      >
+        Stories in AMP - Hello World
+      </a>
+    </amp-story-player>
+    <p>
+      Such treat big ol pupper. Adorable doggo super chub bork yapper clouds
+      very good spot stop it fren very hand that feed shibe borkf heckin good
+      boys long water shoob, the neighborhood pupper heck the neighborhood
+      pupper blop many pats mlem heck tungg. noodle horse. Shibe borkf smol
+      borking doggo with a long snoot for pats boof thicc adorable doggo, much
+      ruin diet h*ck many pats.
+    </p>
+  </body>
+</html>
+```
+[/example]
+
+# Include the Web Story Player
+
+Web Stories rely on the AMP Story Player, in the form of the `<amp-story-player>` element, to embed and play stories on non-AMP sites. Implement the `amp-story-player` element by including the following two scripts in the head of your document:
+
+`<script async src="https://cdn.ampproject.org/amp-story-player-v0.js"></script>`
+
+`<link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel="stylesheet" type="text/css">`
+
+The first script imports the logic for the AMP Story Player. The second script sets the default styling. 
+
+After including each script, include an `<amp-story-player>` element inside the document body.
+
+```html
+<amp-story-player style="width: 370px; height: 622px;">
+    <a
+      href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/">
+      Stories in AMP - Hello World
+    </a>
+  </amp-story-player>
+```
+
+## Size the Web Story Player
+
+You may define the story player's width, height, and other styles inline or as you would any other element's style.
+
+```html
+<body>
+...
+  <amp-story-player style="width: 370px; height: 622px;">
+...
+  </amp-story-player>
+...
+</body>
+```
+
+We recommend maintaining a 69:116 aspect ratio for the best user experience, but you may define any width and height.
+
+
+### Responsive sizing
+
+The story player's responsiveness works as any other block element. Use CSS to maintain width and height ratios, such as the example below:
+html
+```
+<amp-story-player style="width: 50vw; height: 84.05vw;">
+  ...
+</amp-story-player>
+```
+
+## Configure the Web Story Player behavior
+
+The Web Story Player allows you to specify behavior, such as autoplaying stories or circular wrapping. These are specified in the Web Story JSON behavior configuration. 
```

Comment:
I believe "autoplaying stories" might not be what you have in mind.
What it does is (technically) it goes from `prerender` to `visible` when the story gets visible in the user viewport. If the pages don't have an auto-advancement, then the story won't auto-advance.

If the Player is embedded in the page's content, you want `autoplay` to be true, as it will show the story just like when you load a Story through its direct link.
If you want to wait for the user to click on a UI element (play icon, or a card to open a full screen story), then you want to set `autoplay` to false and call `play()` manually upon user interaction.

Hope that clarifies it!